micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/MetricsAspect.groovy

```diff
@@ -41,7 +51,7 @@ class MetricsAspect {
         final URI uri = new URI(uriString)
         final String path = fixSpecialCharacters(uri.path)
         int port = (uri.port > 0)? uri.port : 80
-        return trimDots("${PREFIX}.${uri.host}.${port}.${path}")
+        return sanatizeIpAddressIfPresent(trimDots("${PREFIX}.${uri.host}.${port}.${path}"))
```

Comment:
I'd rather not, if there are many REST resources, we now get nice tree in Graphite. With underscore everything will be flat under one `RestOperations` node. But I agree whole server name (+ maybe port) should be a single hierarchy level.